client/js/views/GameView.vue

```diff
@@ -446,7 +420,7 @@ export default {
     // Game, Deck, Log and Scrap //
     ///////////////////////////////
     game() {
-      return this.$store.state.game;
+      return this.game;
```

Comment:
I believe this still needs the $store reference lest it recurse

```suggestion
      return this.$store.state.game;
```